gneiss/balances.py

```diff
@@ -162,3 +164,99 @@ def _count_matrix(treenode):
             counts[n]['k'] = counts[n.parent]['k'] + counts[n.parent]['r']
             counts[n]['t'] = counts[n.parent]['t']
     return counts, n_tips
+
+
+def sparse_balance_basis(tree):
+    """ Calculates sparse representation of an ilr basis from a tree.
+
+    This computes an orthonormal basis specified from a bifurcating tree.
+
+    Parameters
+    ----------
+    tree : skbio.TreeNode
+        Input bifurcating tree.  Must be strictly bifurcating
+        (i.e. every internal node needs to have exactly 2 children).
+        This is used to specify the ilr basis.
+
+    Returns
+    -------
+    scipy.sparse.coo_matrix
+       The ilr basis required to perform the ilr_inv transform.
+       This is also known as the sequential binary partition.
+       Note that this matrix is represented in clr coordinates.
```

Comment:
Note that this is in clr coordinates, which is the same as the `_balance_basis` function.